### DISCUSSION TITLE
Add Stickers

### DISCUSSION THREAD

**Discussion by Genderfluid** [stoatchat/discussions#94] [Feature Suggestions]
Add stickers.

**Comment by sussycatgirl** [comment]
Oh god please don't make this paid. I hate having to pay on Discord for global custom emojis

**Comment by nikeedev** [comment]
> Add stickers, something Discord has as a boost perk.

Cool feature! But remember to hide where you got the idea from so Mods in Discord doesnt close down Revolt, as discord got harmed already by Google and Youtube.

**Reply by 0xN0x** [reply]
Telegram and Signal had stickers long time before Discord, so it will be ok
